SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -3571,7 +3571,7 @@ \subsection{Code Layout in Tests}
 \item \code{foo} class instantiation and basic object checks (e.g.
 \code{EXERCISE\_BASIC\_OBJECT\_METHODS}).
 \item \code{foo} class properties' input argument read and test (e.g. using the
-macros in \code{itkTestingMacro.h}, such as \code{TEST\_SET\_GET\_OBJECT}, etc.).
```

Comment:
These macros are now prefixed by `ITK_`, so `ITK_TEST_SET_GET_OBJECT` etc. See https://github.com/InsightSoftwareConsortium/ITK/commit/dbb1532dd8fc1d0c263a71ded75944a30a890a6c.

Reply:
Now I see you that are aware of the rename.